docs/network/smart-rollup-nodes.md

```diff
@@ -217,3 +209,52 @@ For example, this query gets the health of the node:
    ```
 
 Now that you have a Smart Rollup node configured for Etherlink, you can run an Etherlink EVM node, as described in [Running an Etherlink EVM node](/network/evm-nodes).
+
+## Stopping the Smart Rollup node
+
+When you want to stop running the Smart Rollup node, you must wait for the last commitment that your node made to be cemented.
+Then you can recover the node's bonded tez and stop the node.
+
+:::warning
+
+If you stop the node before waiting for its last commitment to be cemented, the bonded tez is at risk if other nodes challenge that commitment and your node is not online to defend it in the Smart Rollup refutation game.
+For more information, see [Smart Rollups](https://docs.tezos.com/architecture/smart-rollups) on docs.tezos.com.
+
+:::
+
+Follow these steps to stop an Etherlink Smart Rollup node:
+
+1. Stop the node temporarily.
+
+1. Restart the node in `bailout` mode.
+A node running in `bailout` mode defends its existing commitments but does not make new commitments.
+
+   ```bash
+   octez-smart-rollup-node run bailout for sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf \
+     with operators \
+     operating:$OPERATOR_ACCOUNT \
+     cementing:$OPERATOR_ACCOUNT \
+     recovering:$OPERATOR_ACCOUNT \
```

Comment:
Just to simplify ?
```suggestion
     with operators $OPERATOR_ACCOUNT
```

Reply:
@sribst When I did it with only `operating:$OPERATOR_ACCOUNT`, I got an error that cementing and recovering were required. Maybe "with operators" covers these other cases?

Reply:
The default operator must not be prefixed (`operating:$OPERATOR_ACCOUNT`) but be only the key: `$OPERATOR_ACCOUNT`.

When prefixed (`operating, batching, cementing, recovering, executing_outbox.`) the key is used only for the operations of that prefix